src/plugins/analysis/kernel_config/view/kernel_config.html

```diff
@@ -7,6 +7,31 @@
             <td>Kernel Config</td>
             <td><pre class="border rounded p-2 bg-light" style="white-space: pre-wrap; height: 450px; overflow: scroll;"><code>{{ firmware.processed_analysis[selected_analysis]['kernel_config'] | safe }}</code></pre></td>
         </tr>
+        {% if 'checksec' in firmware.processed_analysis[selected_analysis] and firmware.processed_analysis[selected_analysis]['checksec'] != {} %}
+            <tr>
+                <td>checksec</td>
```

Comment:
checksec is a low-level name for the analysis. It should be named more expressively. Sth. like exploit mitigations as in the original checksec plugin.

Reply:
changed to "Exploit Mitigations"